### PR TITLE
docs(tokens): add token documentation

### LINKED ISSUE #9177: Add usage docs for different entrypoints for token values
Tokens will be able to be brought in through a few different entry points, so we will need to write guidance on how to do this.

- [ ] Color tokens
- [ ] Type tokens
- [ ] Motion tokens

### PR DESCRIPTION
**Pull request by andreancardona**
Closes #9177

- adds a couple more pieces of information for the following packages: 

- [ ] type tokens

- [ ] migration tokens

- [ ] color tokens

### CODE REVIEW

File: packages/styles/docs/sass.md [hunk 1]
```diff
@@ -89,10 +99,23 @@ between version updates.
...
+The motion package provides helper functions, mixins, and duration tokens to add motion into your project.
+
+You can bring in the motion package and tokens by using `@use`. 
+
+For example:
+
+```scss
+ @use '@carbon/styles/scss/motion';
+ .dropdown--arrow {
+  transition: transform $duration-fast-02 motion(standard, productive);
```
Comment (joshblack) [inline packages/styles/docs/sass.md:179]: I think the snippet will need to be updated to:

```scss
@use '@carbon/styles/scss/motion' as *;
```

For those variables to be defined.

File: packages/styles/docs/sass.md [hunk 2]
```diff
@@ -65,10 +65,20 @@ between version updates.
 
 ## Colors
 
+The colors package will give you access to every color token within the IBM Design Language.
```
Comment (joshblack) [inline packages/styles/docs/sass.md]:
```suggestion
The colors package will give you access to each swatch from the IBM Design Language. You can refer to a color by its swatch and grade.
```

File: packages/styles/docs/sass.md [hunk 3]
```diff
@@ -65,10 +65,20 @@ between version updates.
...
+
+Access these color tokens for your project by using `@use`. For example: 
+
+```scss
+ @use '@carbon/styles/scss/colors';
```
Comment (joshblack) [inline packages/styles/docs/sass.md]:
```suggestion
 @use '@carbon/styles/scss/colors';
 
 .selector {
   background-color: colors.$blue-50;
 }
```

File: packages/styles/docs/sass.md [hunk 4]
```diff
@@ -65,10 +65,20 @@ between version updates.
...
+```
+
 | Import                               | Filepath            |
```
Comment (joshblack) [inline packages/styles/docs/sass.md:82]: Make sure this is before all the content above

File: packages/styles/docs/sass.md [hunk 5]
```diff
@@ -65,10 +65,20 @@ between version updates.
...
 | :----------------------------------- | :------------------ |
 | `@use '@carbon/styles/scss/colors';` | `scss/_colors.scss` |
 
+For more information checkout our [Files](#files) docs.
```
Comment (joshblack) [inline packages/styles/docs/sass.md]: To see all the colors available to be imported, checkout our [colors](#todo) docs.

File: packages/styles/docs/sass.md [hunk 6]
```diff
@@ -89,10 +99,23 @@ between version updates.
 
 ## Motion
 
+The motion package provides helper functions, mixins, and duration tokens to add motion into your project.
+
+You can bring in the motion package and tokens by using `@use`. 
+
+For example:
+
```
Comment (joshblack) [inline packages/styles/docs/sass.md]: I think we can drop these kinds of snippets and just reference the snippet directly 👀 

```suggestion
```

File: packages/styles/docs/sass.md [hunk 7]
```diff
@@ -89,10 +99,23 @@ between version updates.
...
+ @use '@carbon/styles/scss/motion';
+ .dropdown--arrow {
+  transition: transform $duration-fast-02 motion(standard, productive);
+ }
+```
 | Import                               | Filepath            |
 | :----------------------------------- | :------------------ |
 | `@use '@carbon/styles/scss/motion';` | `scss/_motion.scss` |
 
+For more information checkout our [Files](#files) docs.
```
Comment (joshblack) [inline packages/styles/docs/sass.md]:
```suggestion
For more information, checkout our [motion](#todo) docs.
```

File: packages/styles/docs/sass.md [hunk 8]
```diff
@@ -187,10 +210,19 @@ options:
 
 ## Type
 
+The type package entrypoint allows you to specifically bring type tokens into your project. The type package includes various type tokens and mixins.
+
+To bring in the `@carbon/type` package use `@use`. For example:
```
Comment (joshblack) [inline packages/styles/docs/sass.md]:
```suggestion
```

File: packages/styles/docs/sass.md [hunk 9]
```diff
@@ -187,10 +210,19 @@ options:
 
 ## Type
 
+The type package entrypoint allows you to specifically bring type tokens into your project. The type package includes various type tokens and mixins.
+
+To bring in the `@carbon/type` package use `@use`. For example: 
+
+```scss
+ @use '@carbon/styles/scss/type';
```
Comment (joshblack) [inline packages/styles/docs/sass.md]:
```suggestion
 @use '@carbon/styles/scss/type';
 
 .selector {
   @include type.style(type.$productive-heading-01);
 }
```

File: packages/styles/docs/sass.md [hunk 10]
```diff
@@ -187,10 +210,19 @@ options:
...
+```
+
 | Import                             | Filepath          |
 | :--------------------------------- | :---------------- |
 | `@use '@carbon/styles/scss/type';` | `scss/_type.scss` |
 
+For more information checkout our [Files](#files) docs.
```
Comment (joshblack) [inline packages/styles/docs/sass.md]:
```suggestion
For more information, check out our [type](#todo) docs.
```

**Review by joshblack — CHANGES_REQUESTED** [review]

**Review by joshblack — APPROVED** [review]

**Review by sstrubberg — APPROVED** [review]

**Pull request merged**